### ISSUE TITLE
Suggestion: Prevent users from updating mods from a modpack

### ISSUE THREAD

**Issue by RuinTD** [PrismLauncher/PrismLauncher#440]
### Role

I play modded Minecraft

### Suggestion

The Check for Updates button in an instance's Mods menu should (by default) skip mods installed from a modpack

### Benefit

Prevents users from accidentally breaking a modpack

### This suggestion is unique

- [X] I have searched the issue tracker and did not find an issue describing my suggestion, especially not one that has been rejected.

### You may use the editor below to elaborate further.

Maybe there could be an option when clicking Check for Updates prompting the user if they want to check for modpack updates; with the option to skip, check for modpack updates, or force update the mods.
BTW, mods added by the user should be updatable.

**Comment by TheAirBlow** [comment]
I agree. A player may think nothing about it, but it would actually break a lot of quests and CraftTweaker scripts.

**Comment by Poopooracoocoo** [comment]
Where should the options you mention go? Got a sketch?
